avaframe/in3Utils/cfgUtils.py

```diff
@@ -630,6 +635,35 @@ def convertDF2numerics(simDF):
     return simDF
 
 
+def setStrnanToNan(simDF, simDFTest, name):
+    """ set pandas element to np.nan if it is a string nan
+
+        Parameters
+        -----------
+        simDF: pandas dataFrame
+            dataframe
+        simDFTest: pandas series
+            series of sim DF column named name
+            replaced "." with " "
+        name: str
+            name of pandas dataframe column
+
+        Returns
+        --------
+        simDF: pandas dataframe
+            updated pandas dataframe with np.nan values where string nan was
+    """
+
+    nanIndex = simDFTest.str.match('nan')
```

Comment:
Should we lower or set the IGNORECASE?
https://pandas.pydata.org/docs/reference/api/pandas.Series.str.match.html

Reply:
I added flag re.IGNORECASE